sdk/tables/azure-data-tables/tests/test_table_entity_async.py

```diff
@@ -1723,7 +1723,7 @@ async def test_sas_signed_identifier(self, resource_group, location, storage_acc
                 self.account_url(storage_account, "table"),
                 credential=token,
             )
-            table = service.get_table_client(table=self.table_name)
+            table = service.get_table_client(table_name=self.table_name)
```

Comment:
No, I'm not sure how this got through, the parameter has been called `table_name` for at least a month and this has just slipped under the radar. Probably because the live pipeline hasn't been up and running